Dockerfile

```diff
@@ -6,16 +6,16 @@ WORKDIR /app
 
 # copy package and lock files first for better caching
 COPY ./package.json /app/package.json
-COPY ./yarn.lock /app/yarn.lock
+COPY ./pnpm-lock.yaml /app/pnpm-lock.yaml
 
 # install the packages
-RUN yarn install --frozen-lockfile --ignore-scripts
+RUN pnpm install --frozen-lockfile --ignore-scripts
```

Comment:
We need to make sure this actually work, I think `pnpm` is not installed by default on `node` images.